src/OpenTelemetry.Instrumentation.ServiceFabricRemoting/OpenTelemetry.Instrumentation.ServiceFabricRemoting.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>ServiceFabric Remoting instrumentation for OpenTelemetry .NET.</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.ServiceFabricRemoting-</MinVerTagPrefix>
+    <PackageValidationBaselineVersion>1.9.0</PackageValidationBaselineVersion>
```

Comment:
I think that you do not have any baseline to validation. It should be disabled as long as you do not have stable release. You can find examples in most of the instrumentation libraries how to do this.

This section

```
  <!-- Do not run Package Baseline Validation as this package has never released a stable version.
  Remove this property once we have released a stable version and add PackageValidationBaselineVersion property. -->
  <PropertyGroup>
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
  </PropertyGroup>
```

Reply:
fixed